### ISSUE TITLE
Can this semantic segmentation process be visualized?please details the related settings about Apollo

### ISSUE THREAD

**Issue by bobododosjl** [ShanWang-Shan/HomoFusion#4]

**Comment by ShanWang-Shan** [comment]
Thank you for your interest in our work. I’m not entirely sure I fully understand what you mean by "visualize process." The segmentation results are displayed in color, as shown in the paper. Most of our settings can be found in the YAML files. For the specific settings regarding Apollo, please refer to: config/data/apolloscape.yaml.

**Comment by bobododosjl** [comment]
can this segmentation results been showned by running your open source code



---Original---
From: ***@***.***&gt;
Date: Sun, Sep 29, 2024 08:14 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [ShanWang-Shan/HomoFusion] Can this semantic segmentation processbe visualized?please details the related settings about Apollo (Issue #4)




 
Thank you for your interest in our work. I’m not entirely sure I fully understand what you mean by "visualize process." The segmentation results are displayed in color, as shown in the paper. Most of our settings can be found in the YAML files. For the specific settings regarding Apollo, please refer to: config/data/apolloscape.yaml.

**Comment by ShanWang-Shan** [comment]
Yes, the segmentation results are generated in PNG format with color-coded segmentation.

**Comment by bobododosjl** [comment]
ok

**Comment by bobododosjl** [comment]
Can this method be integrated into SLAM framework?

**Comment by ShanWang-Shan** [comment]
We have not tried this yet. However, I believe it is possible to use our segmentation as an additional feature extractor to supplement SLAM for matching.

**Comment by bobododosjl** [comment]
thank u

**Comment by bobododosjl** [comment]
[lane_marking_examples.tar.gz](https://ad-apolloscape.cdn.bcebos.com/lane_marking_examples.tar.gz) can this dataset be used to test?

**Comment by ShanWang-Shan** [comment]
Our model was trained on the Apolloscape dataset. If your data is also from this dataset and you have the camera extrinsics, the data can be used for testing. However, if the data comes from a different source, fine-tuning with some of the new data would be necessary.

**Comment by bobododosjl** [comment]
I have already download checkpoint of Apollo, when I run 'python3 scripts/benchmark_val.py', it download "https://github.com/lukemelas/EfficientNet-PyTorch/releases/download/1.0/efficientnet-b6-c76e70fd.pth", is that right?

**Comment by ShanWang-Shan** [comment]
It is correct, the original training starts from EfficientNet's pre-trained weights. Initially, it downloads the EfficientNet pre-trained weights, and then it loads our specific pre-trained weights.

**Comment by bobododosjl** [comment]
I download the  EfficientNet's pre-trained weights, but I still meet the error.
[Uploading log.txt…]()

**Comment by ShanWang-Shan** [comment]
I was not able to open your error file

**Comment by bobododosjl** [comment]
I have sent to your email

**Comment by bobododosjl** [comment]
Is data from lane_segmentation of Apollo dataset?

**Comment by ShanWang-Shan** [comment]
Yes, the data is from the lane_segmentation of the Apollo dataset, and I have updated the README to clarify this. I have also included a link to download the pose information we used, which comes from the Apollo self-localization dataset.
